### PR TITLE
system-tools: add inotify-tools, mc

### PR DESCRIPTION
**Pull request by awiouy**
mc builds, I have not tested it, though

### CODE REVIEW

**Comment by chewitt** [comment]
let me know when it's been tested then :)

**Comment by awiouy** [comment]
It runs RPi2 here ;)

**Comment by awiouy** [comment]
Do not merge, yet!

**Comment by awiouy** [comment]
Rebased to replace tools.system-tools with virtual.system-tools.
Please merge

**Comment by TheUlpio** [comment]
Why don't add an option to choose between nano and mc in options? Both have edit capabilities and image might be (a bit) slighter

**Comment by awiouy** [comment]
nano comes with LE and is officially supported. mc comes with the optional system-tools addon which has discretionary support.

**Comment by awiouy** [comment]
Rebased after #708 
Ready for review/merge

**Comment by awiouy** [comment]
Rebased to add inotify-tools

File: packages/addons/addon-depends/system-tools-depends/inotify-tools/package.mk [hunk 1]
```diff
@@ -0,0 +1,21 @@
+PKG_NAME="inotify-tools"
```
Comment (chewitt) [inline packages/addons/addon-depends/system-tools-depends/inotify-tools/package.mk]: ^^ no LE license header

Comment (awiouy) [inline packages/addons/addon-depends/system-tools-depends/inotify-tools/package.mk]: Done

**Pull request merged**